### PR TITLE
Add maxFanout to galley's options

### PR DESCRIPTION
**Pull request by tiago-loureiro**
This will be required for https://github.com/wireapp/wire-server/pull/1060 to pass the tests

### CODE REVIEW

File: charts/galley/templates/configmap.yaml [hunk 1]
```diff
@@ -43,6 +43,7 @@ data:
       intraListing: false
       maxTeamSize: {{ .settings.maxTeamSize }}
       maxConvSize: {{ .settings.maxConvSize }}
+      truncationLimit: {{ .settings.truncationLimit }}
```
Comment (akshaymankar) [inline charts/galley/templates/configmap.yaml:46]: It will make upgrades easier if we default this to 500 like the `maxTeamSize`.

Comment (tiago-loureiro) [reply charts/galley/templates/configmap.yaml:46]: Actually this is optional, and in general nobody should set it. Maybe we can adjust it to:
```
{{- if .settings. truncationLimit }}
    truncationLimit: {{ .settings.truncationLimit }}
{{- end }}
```

Comment (akshaymankar) [reply charts/galley/templates/configmap.yaml:46]: Yeah, that is even better.

Comment (tiago-loureiro) [reply charts/galley/templates/configmap.yaml:47]: https://github.com/wireapp/wire-server-deploy/pull/231/commits/2c4a00d6854a48074f812fa521ffe81351522f86

**Comment by tiago-loureiro** [comment]
`maxFanoutSize` is indeed a better name

**Pull request merged**